indico/modules/events/editing/__init__.py

```diff
@@ -14,12 +14,16 @@
 from indico.core.permissions import ManagementPermission
 from indico.modules.events.features.base import EventFeature
 from indico.modules.events.models.events import Event, EventType
+from indico.modules.events.settings import EventSettingsProxy
 from indico.util.i18n import _
 from indico.web.flask.util import url_for
 from indico.web.menu import SideMenuItem
 
 
 logger = Logger.get('events.editing')
+editing_settings = EventSettingsProxy('editing', {
```

Comment:
I'd move this to a `settings.py` (we did the same in many other modules except the early ones)